backend/index.js

```diff
@@ -51,4 +55,8 @@ app.get('/my-counter', (req, res) => {
 
 app.get('/env', (req, res) => res.status(200).send(`ENV: ${process.env.ENV}`));
 
+app.get('*', function (req, res) {
+  res.sendFile(path.join(process.cwd(), FE_BUILD_PATH, 'index.html'));
```

Comment:
Change function to arrow function please.
Also please try to remove process.cwd() as FE_BUILD_PATH/index.html should already be what you want.